controllers/provisioning_controller.go

```diff
@@ -159,13 +159,13 @@ func (r *ProvisioningReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error
 	}
 
 	//Create Secrets needed for Metal3 deployment
-	if err := provisioning.CreateMariadbPasswordSecret(r.kubeClient.CoreV1(), ComponentNamespace); err != nil {
+	if err := provisioning.CreateMariadbPasswordSecret(r.KubeClient.CoreV1(), ComponentNamespace); err != nil {
```

Comment:
If the client wasn't configured at all before, how were these calls working in the test suite?

Reply:
We initialize them as Fake clients in the test suite.

Reply:
This was an oversight on my end. I knew it had to be done but somehow missed it the the commit.